### PR TITLE
Added the ability to view XCom variables in webserver

### PR DESCRIPTION
**Pull request by withnale**
Screenshot speaks a thousand words.

screen shot 2016-04-08 at 09 23 42

NB: If you don't wish the XCom variables to appear, mark them as private by prefixing them with _

### CODE REVIEW

**Comment by bolkedebruin** [comment]
Thanks! Can you add a license header to the new files you added? This is required for our apache incubator status.

File: airflow/www/templates/airflow/xcom.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{% extends "airflow/task_instance.html" %}
+{% block title %}Airflow - DAGs{% endblock %}
+
+{% block body %}
+    {{ super() }}
+    <h4>{{ title }}</h4>
+    <div>
+
+        <h5>Variables</h5>
```
Comment (r39132) [inline airflow/www/templates/airflow/xcom.html]: You may not need the "Variables" subheading here - does it add anything?

"Variables" may also refer to Airflow Variables -- Airflow Variables is a bit of an unfortunate name.

Comment (jlowin) [inline airflow/www/templates/airflow/xcom.html]: Good point... Airflow Variables are sort of like global XComs.

**Comment by r39132** [comment]
Thx. @withnale Please squash your commits into a single one and ping me on gitter to merge.

File: airflow/www/templates/airflow/xcom.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            {% for attr, value in attributes %}
+                <tr>
+                    <td>{{ attr }}</td>
+                    <td class='code'>{{ value }}</td>
+                </tr>
+            {% endfor %}
+        </table>
+        {{ html_code|safe }}
+    </div>
+{% endblock %}
```
Comment (r39132) [inline airflow/www/templates/airflow/xcom.html:37]: This file needs an apache license header too. Please add and squash commits.

**Comment by r39132** [comment]
@withnale You need to also add an apache license header on top of the xcom.html file. Please fix, squash commits, and ping me on gitter when ready. 

You can follow the example on https://cwiki.apache.org/confluence/display/WW/xhtml+theme

**Comment by r39132** [comment]
:100:

**Comment by r39132** [comment]
Thx for this change @withnale

**Comment by jlowin** [comment]
:+1:

**Comment by mistercrunch** [comment]
Nice!

**Pull request merged**